.github/workflows/release.yml

```diff
@@ -0,0 +1,148 @@
+# TODO: consider updating requirements.txt for the documentation step,
+#       as it does not work with python versions >3.8
```

Comment:
I now updated Sphinx in the requirements.txt, is this now correct?
The build now seems to be giving a green checkmark.

Reply:
it does not build sphinx though, both pr and current workflow only triggered docs build for tags
should it be included into pr step as well?

Reply:
Not sure, as the RTD server also builds the docs.
I have no idea how popular the included docs are for the nightly builds.